docker/continuous-integration/Dockerfile

```diff
@@ -193,6 +193,15 @@ RUN curl -SsL https://github.com/boxboat/fixuid/releases/download/v0.4/fixuid-0.
 # install pdm for managing our python dependencies
 RUN pip install --system --prefix=/usr/local pdm
 
+# install sc, the sauce connect proxy for running UI tests
+RUN mkdir sauce_connect && \
```

Comment:
Is there a particular reason to do this in a temp directory rather than in `/opt/sauce-connect` as recommended by [the generic linux installation docs](https://docs.saucelabs.com/secure-connections/sauce-connect-5/installation/linux/#generic)? I don't see any _problem_ with doing it like this, just curious.

Reply:
no particular heavy thought, I ended up mimicking how we install other binaries in the docker vs mimicking their directions (which seemed to be complicated that way mainly because they want a dir for their shell completion scripts, etc)

Reply:
Ahh, that makes sense. Thanks!